src/System.Private.CoreLib/shared/System/Security/VerificationException.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
@safern has a pending PR to turn on nullability for all exceptions.  Does that PR not cover this file?

Reply:
I don't think so (I asked him not to change him) - will revert if it does